photocentric add on gcode control

kloknibor opened the pull request.

WesGilster commented:
I'm totally fine with this:
host/src/org/area515/resinprinter/gcode/PhotocentricGCodeControl.java

However, you've also changed:
host/src/org/area515/resinprinter/gcode/GCodeControl.java

...and this affects all other printers. I'm not sure I'm comfortable with that, plus it totally invalidates the need to have a PhotocentricGCodeControl class.

It's also hard to determine what changes you made to the javascript files due to the way github manged the +- deltas.

kloknibor commented:
Oh shit... I did indeed some other changes because I wanted annquick edit to test... Didn't realise they would end up here... Sorry... I'll reapply with only the driver thing

kloknibor commented:
(The things I had changed in here are accidentally the Gcodecontrol.java this shouldn't have been done, I addes photcentricGcodecontrol.java, this should be done, I deleted X-Y travel, Projector on/off and shutter) But I did an new PR
